cmd/kubeadm/app/cmd/util/markdown.go

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
This is code copied from kubectl right?
Can't we just make a generic `pkg/utils/` package for this that both commands consume instead?